app/helpers/patternfly_components_helper.rb

```diff
@@ -23,21 +23,20 @@ def title_tag(title)
     end
   end
 
-  def body_tag(body)
-    tag.div class: 'pf-c-alert__description' do
-      tag.p body
-    end
-  end
+  def description_tag(description)
+    return unless description
 
-  def pf_inline_alert(title, body, variant: nil)
-    tag.div class: "pf-c-alert pf-m-#{variant} pf-m-inline" do
-      icon_tag(variant) + title_tag(title) + body_tag(body)
+    tag.div class: 'pf-c-alert__description' do
+      tag.p description
     end
   end
 
-  def pf_inline_alert_plain(title, variant: nil)
-    tag.div class: "pf-c-alert pf-m-#{variant} pf-m-inline pf-m-plain" do
```

Comment:
if `variant` is `nil`, the class `pf-m-#{variant}` would be `pf-m-` which probably wouldn't make harm, but I decided to remove it.